doc/how-to-handle-a-release-on-github.md

```diff
@@ -0,0 +1,24 @@
+# How to Handle a Release on Github
+
+The release tools are aligned to the [cactus branch model](https://barro.github.io/2016/02/a-succesful-git-branching-model-considered-harmful/cactus-model.pdf) approach. This means you merge your pull requests to your integration branch (master). If you want to make a new release, you create a new release branch based on master (or a tag) and then make this branch ready for the deployment. From this time on master and the release branch are diverging. No merging from one to the other branch happens.
+
+## Example 1: Standard Release
+
+If you want to create a new release based on the master branch
+
+![visualizing a release on github](./how-to-handle-a-release.jpg)
+
+1. `li-release init-release 1.0.0 master` creates a new release branch `release-1.0.0` based on the last master commit
+2. make your fixes to the release branch `release-1.0.0` until it's ready for a deployment
+3. `li-release finish-release 1.0.0` adds a tag to the last commit in the branch `release-1.0.0`
+4. Now you are ready for a deployment
```

Comment:
Just a remark:
- `create-release` or `create-release-branch` instead of `init-release`
- `tag-release` or `tag-release-branch` instead of `finish-release`

Because it says so in the descriptions of the commands.